grammars/php.cson

```diff
@@ -1679,6 +1651,7 @@
       }
       {
         'match': '''(?x)
+          (^\\s*)?
```

Comment:
Normally I would be against including something like this because it effectively moves this pattern to the top of the stack but given the circumstances I think it's ok.